pkg/service/operation/credential/response.go

```diff
@@ -0,0 +1,35 @@
+package credential
```

Comment:
this file is basically the same as the presentation/response file right? (except the parentResource)

Is there anyway we can combine, or do you want it to be separate so they can implement different status if they want to

Reply:
Status is already slightly different: rejected vs denied, fulfilled vs approved. 

I think there is a chance of unifying, but I'm not comfortable yet deciding what the correct generalization would look like.